docker-compose.yml

```diff
@@ -237,6 +237,8 @@ services:
     environment:
       VIRTUAL_HOST: "mail${DOMAIN_SUFFIX}"
       VIRTUAL_PORT: 8025
+    ports:
```

Comment:
Should already work if you add `127.0.0.1 mail.local` to your /etc/hosts through being proxied by the nginx-proxy, so we don't need to block more ports on the host system

Reply:
But that should be documented I guess :wink:

Reply:
I removed the port and updated the doc